### PR TITLE
Remove unneeded values from Delete call for `iot_dps_certificate`

### PR DESCRIPTION
**Pull request by mbfrahry**
It looks like these values are unneeded in the Delete call for the IoT Device Provisioning Service Certificate. Would it make sense to remove these values?

### CODE REVIEW

**Comment by amarzavery** [comment]
@mbfrahry - The CI is failing. You need to update x-ms-examples by removing the parameter values for removed parameters. All the parameters are being removed from a stable version of the swagger spec. If they were unnecessary then why were they added in the first place. This will cause breaking changes in the sdk.

File: specification/deviceprovisioningservices/resource-manager/Microsoft.Devices/stable/2018-01-22/iotdps.json [hunk 1]
```diff
@@ -268,73 +268,6 @@
             "type": "string",
             "description": "This is a mandatory field, and is the logical name of the certificate that the provisioning service will access by."
           },
```
Comment (ravbhatnagar) [inline specification/deviceprovisioningservices/resource-manager/Microsoft.Devices/stable/2018-01-22/iotdps.json]: why were these query parameters added to delete? They are not present on the PUT. Were these somehow supported on the service side?

**Review by ravbhatnagar — COMMENTED** [review]
@mbfrahry - can you please shed some light on why were they present on the delete API?

**Comment by yungezz** [comment]
hi @mbfrahry  are you still on this PR? could  you pls address the comments from ARM, and fixing CI failure per @amarzavery  suggestion? thanks.

**Comment by yungezz** [comment]
@mbfrahry is it ok to close the PR for now since you're not able to address comment? the PR is over several months. Pls create new PR when you're ready for review and merge.

**Comment by yungezz** [comment]
close for now since no update from author.

**Pull request closed without merging**